packages/amplify-graphql-api-construct/API.md

```diff
@@ -130,6 +131,7 @@ export interface AmplifyGraphqlApiProps {
     readonly definition: IAmplifyGraphqlDefinition;
     readonly functionNameMap?: Record<string, IFunction>;
     readonly functionSlots?: FunctionSlot[];
+    readonly logging?: true | LogConfig;
```

Comment:
We need to make sure this gets propagated to `@aws-amplify/data-construct` as well, so it can be used by Gen 2 schema builder.

Reply:
I suppose we need to propogate `logging`, not `LogConfig` right?

Reply:
propagated both